blazor/treeview/navigation.md

```diff
@@ -0,0 +1,333 @@
+---
+layout: post
+title: Navigation in Blazor TreeView Component | Syncfusion
+description: Checkout and learn here all about navigation in Syncfusion Blazor TreeView component and much more.
+platform: Blazor
+control: TreeView
+documentation: ug
+---
+
+# Navigation URL binding in Blazor TreeView Component
+
+The TreeView component's **NavigateUrl** property is used to navigate from one page to other pages with TreeView node selection.
+
+In the following example, the `NavigateUrl` property is enabled.

```

Comment:
In the following example, the `NavigateUrl` field is mapped.

Reply:
Done.

Reply:
Done.